core/src/main/resources/reference.conf

```diff
@@ -149,7 +149,7 @@ backend {
         run-in-background = true
         runtime-attributes = "String? docker"
         submit = "/bin/bash ${script}"
-        submit-docker = "docker run --rm -v ${cwd}:${docker_cwd} -i ${docker} /bin/bash < ${script}"
+        submit-docker = "docker run --rm -v ${cwd}:${docker_cwd} -i ${docker} /bin/bash ${docker_cwd}/execution/script"
```

Comment:
When the command runs inside the container, the path to the script is the latter.